packages/dht/.gitignore

```diff
@@ -0,0 +1,13 @@
+.DS_Store
+node_modules
+lerna-debug.log
+*.tsbuildinfo
+*.heapprofile
+*.heapsnapshot
+oldsrc
+dist
+google
+src/proto
```

Comment:
Is it possible to put the generated code outside `src` directory? E.g. to `generated/`